docker-compose.yml

```diff
@@ -27,20 +26,16 @@ services:
     ports:
     - 0.0.0.0:$ELO_WWW_PORT:81
     networks:
-      elonet:
-        ipv4_address: 172.16.11.3
+      - elonet
   elo_db:
     container_name: elo_db
     image: mysql:5.7
-    ports:
-    - 127.0.0.1:$DB_PORT:3306
```

Comment:
Do bazy danych nie łączymy się z zewnątrz, więc nie ma potrzeby wystawiać portu.

Reply:
Ja wolę korzystać z zewnętrznego oprogramowania, więc to nie jest do usunięcia...

Reply:
Racja, o tym nie pomyślałem.